packages/terra-folder-tree/src/FolderTree.jsx

```diff
@@ -13,37 +18,84 @@ const propTypes = {
    * List of FolderTree.Items to be displayed as content within the FolderTree.
    */
   children: PropTypes.node,
+  /**
+   * The heading level for the title of the folder tree.
+   */
+  headerLevel: PropTypes.number,
   /**
    * The title of the folder tree.
    */
   title: PropTypes.string.isRequired,
   /**
-   * The heading level for the title of the folder tree.
+   * Callback function for expand all event.
    */
-  headerLevel: PropTypes.number,
+  onExpandAll: PropTypes.func,
+  /**
+   * Callback function for collapse all event.
+   */
+  onCollapseAll: PropTypes.func,
+  /**
+   * @private
+   * intl object programmatically imported through injectIntl from react-intl.
+   * */
+  intl: PropTypes.shape({ formatMessage: PropTypes.func }).isRequired,
 };
 
 const defaultProps = {
   headerLevel: 3,
 };
 
-const FolderTree = ({ children, title, headerLevel }) => (
-  <>
-    <ActionHeader
-      text={title}
-      level={headerLevel}
-    />
-    <ul
-      className={cx('folder-tree')}
-      role="tree"
-      aria-label={title}
-    >
-      {children}
-    </ul>
-  </>
-);
+const FolderTree = ({
+  children,
+  headerLevel,
+  title,
+  onExpandAll,
+  onCollapseAll,
+  intl,
+}) => {
+  const folderTreeID = `folder-tree-${uuidv4()}`;
+
+  return (
+    <>
```

Comment:
Don't you want a container div around your control?  Wouldn't it be used to control things like having 100% width/height of the parent container?  Might not be the scope for this PR, but also questioning if the toolbar and action header should be sticky and allow the list contents to scroll.

Reply:
Doesn't seem to make a big difference with the current examples but we probably do want a container div, updated: [21394fa](https://github.com/cerner/terra-framework/pull/1943/commits/21394fa7a6498b30ac81df666a4c9513f74e04f5)

I haven't seen anything about the toolbar and action header being sticky. @eawww Do you know if that's something we would want to add?